pkg/shell/base_index.js

```diff
@@ -37,6 +37,11 @@ function Frames(index, setupIdleResetTimers) {
     if (!language)
         language = "en-us";
 
+    var lang_dir = "ltr";
```

Comment:
Can you please use `let` instead of `var`?

Reply:
I was using var like the other variables directly above in the same function. Should those also be changed?